tests/python/unittest/test_numpy_interoperability.py

```diff
@@ -1832,7 +1832,7 @@ def test_scalar_output():
             v2 = vec2.astype(dt)
             OpArgMngr.add_workload('matmul', v1, v2)
             OpArgMngr.add_workload('matmul', v2.T, v1)
-
+    
```

Comment:
Why add the whitespace here (and many other places in this file).

Reply:
git merging issue, will delete